tests/sentry/workflow_engine/endpoints/validators/test_base_action.py

```diff
@@ -84,3 +86,24 @@ def test_validate_data__invalid(self, mock_action_handler_get):
 
         result = validator.is_valid()
         assert result is False
+
+    def test_validate_type__action_gated(self, mock_action_handler_get):
+        organization = Organization(slug="test-org")
```

Comment:
This isn't "wrong" but usually we'd just do `self.organization` (included in the test base class) or if you need to create a new one it'd be `organization = self.create_organization(slug="test-org")`